ui/src/components/admin/fundraise/AdminFundraiseContainer.js

```diff
@@ -0,0 +1,326 @@
+            <FormControl isRequired isInvalid={campaignNameCheck()}>
+              <FormLabel>Campaign Name</FormLabel>
+              <Input
+                type="text"
+                value={campaignName}
+                onChange={(e) => setCampaignName(e.target.value)}
+              />
+              <FormErrorMessage>
+                Another fundraising campaign with the same name already exists
+              </FormErrorMessage>
+            </FormControl>
+            <FormControl isRequired>
+              <FormLabel>Category Name</FormLabel>
+              <Input
+                type="text"
+                value={categoryName}
+                onChange={(e) => setCategoryName(e.target.value)}
+              />
+            </FormControl>
+            <FormControl isRequired>
+              <FormLabel>Category Key</FormLabel>
+              <Input
+                type="text"
+                value={categoryKey}
+                onChange={(e) => setCategoryKey(e.target.value)}
+              />
+            </FormControl>
+            <FormControl isRequired>
+              <FormLabel>Amount</FormLabel>
+              <NumberInput>
+                <NumberInputField
+                  value={amount}
+                  onChange={(e) => setAmount(e.target.value)}
+                />
+              </NumberInput>
+            </FormControl>
+            <FormControl isRequired>
+              <FormLabel>Givers</FormLabel>
+              <NumberInput>
+                <NumberInputField
+                  value={givers}
+                  onChange={(e) => setGivers(e.target.value)}
+                />
+              </NumberInput>
+            </FormControl>
+            <Text></Text>
+            <Box borderWidth="1px" borderRadius="xl" m="2" py="3">
+              <VStack spacing={4} align="start" m="2">
+                <FormControl>
+                  <FormLabel>Milestone Name</FormLabel>
+                  <Input
+                    type="text"
+                    value={milestoneName}
+                    onChange={(e) => setMilestoneName(e.target.value)}
+                  />
+                </FormControl>
+                <FormControl>
+                  <FormLabel>Milestone Amount</FormLabel>
+                  <NumberInput>
+                    <NumberInputField
+                      value={milestoneAmount}
+                      onChange={(e) => setAmount(e.target.value)}
+                    />
+                  </NumberInput>
+                </FormControl>
+                <FormControl>
+                  <FormLabel>Milestone Deadline</FormLabel>
+                  <Input
+                    type="datetime-local"
+                    value={milestoneDeadline}
+                    onChange={(e) => setMilestoneDeadline(e.target.value)}
+                  />
+                </FormControl>
+                <Button colorScheme="blue" onClick={handleAddMilestones}>
+                  Add Milestone
```

Comment:
How do you add the existing milestones ya? I assumed you used the api directly? cuz I cant add the milestone from here.

Reply:
was a bug due to <NumberInput> switched to <Input>